.github/workflows/release.yml

```diff
@@ -1,69 +1,68 @@
 name: Release
 
 on:
-  workflow_dispatch:
-    # Inputs the workflow accepts.
-    inputs:
-      ref:
-        description: 'Commit Reference'
-        default: 'master'
-        required: true
-      tag:
-        description: 'Version Tag'
-        default: 'v0.0.0'
-        required: true
+    workflow_dispatch:
+        # Inputs the workflow accepts.
+        inputs:
+            ref:
+                description: 'Commit Reference'
+                default: 'master'
+                required: true
+            tag:
+                description: 'Version Tag'
+                default: 'v0.0.0'
+                required: true
 
 jobs:
-  build-and-release:
-    runs-on: ubuntu-latest
-    permissions:
-      contents: write
-    steps:
-      - name: Install Go
-        uses: actions/setup-go@v2
-        with:
-          go-version: 1.21.x
+    build-and-release:
+        runs-on: ubuntu-latest
+        permissions:
+            contents: write
+        steps:
+            - name: Install Go
+              uses: actions/setup-go@v2
+              with:
+                  go-version: 1.21.x
 
-      - name: Install Protoc
-        uses: arduino/setup-protoc@v1
-        with:
-          version: 3.x
-          repo-token: ${{ secrets.GITHUB_TOKEN }}
+            - name: Install Protoc
+              uses: arduino/setup-protoc@v1
+              with:
+                  version: 3.x
+                  repo-token: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Install Protoc Go plugin
-        run: go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
+            - name: Install Protoc Go plugin
+              run: go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
 
-      - name: Install Node
-        uses: actions/setup-node@v3
-        with:
-          node-version: '19'
+            - name: Install Node
+              uses: actions/setup-node@v3
+              with:
+                  node-version: 'lts/*'
```

Comment:
This should be changed to 20, not LTS since everything locally is now documented as 20